lib/src/components/NxList/__tests__/NxList.test.tsx

```diff
@@ -190,17 +186,47 @@ describe('NxList', function() {
     expect(component.find(NxLoadError)).toHaveProp('retryHandler', retryHandler);
   });
 
-  it('does not show the emptyMessage when isLoading', async function() {
-    const component = await mountAttached(<NxList emptyMessage="Empty message" isLoading />);
-    expect(component).not.toHaveText('Empty message');
-  });
+  it('renders children correctly', async function() {
+    const children = [
+      <NxList.Item key="1">
+        <NxList.Text>Test Text</NxList.Text>
+        <NxList.Subtext>Test Subtext</NxList.Subtext>
+      </NxList.Item>,
+      <NxList.ButtonItem key="3">
+        <NxList.Text>Test2 ButtonItem Text</NxList.Text>
+        <NxList.Subtext>Test2 ButtonItem Subtext</NxList.Subtext>
+      </NxList.ButtonItem>,
+      <NxList.LinkItem key="4" href="www.sonatype.com">
+        <NxList.Text>Test2 LinkItem Text</NxList.Text>
+        <NxList.Subtext>Test2 LinkItem Subtext</NxList.Subtext>
+      </NxList.LinkItem>
+    ];
+
+    const contentEl = getMounted({children});
```

Comment:
This works but is overkill.  You can just shallow render, passing any old junk as the children, and then confirm that the shallow render's children are reference-equal to the children you passed in

Reply:
> This works but is overkill. You can just shallow render, passing any old junk as the children, and then confirm that the shallow render's children are reference-equal to the children you passed in

Test case has been simplified.
2650adb6c32cf576abb678284230188520340603